backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/json/JsonPatchHttpMessageConverter.java

```diff
@@ -20,26 +20,29 @@ public JsonPatchHttpMessageConverter() {
 		super(MediaType.APPLICATION_JSON, JsonPatchMediaTypes.JSON_PATCH);
 	}
 
+	@SuppressWarnings("null")
```

Comment:
A less-cluttery way to do this would be to move this annotation to the class

Reply:
My first choice is to use `SupressWarnings` at as low a level as possible lest warnings be missed at on some new code added in future.

But I recognize that is unlikely to be a problem here, so implemented in commit 7765171.